### PR TITLE
Remove unused pixel tracking

### PR DESCRIPTION
**Pull request by jonathanKingston**
**Reviewer:** @kdzwinel 

## Description:

Removes all tracking pixels other than the site report as it was all unused.

I renamed the class and hard coded the pixel name so it can't be used again without reimplementing it as it breaks addon policy.

https://app.asana.com/0/892838074342800/1200747843240318/f

## Steps to test this PR:

1. Open up the browser inspector
2. Report a broken site
3. Ensure the report is firing correctly

## Automated tests:
- [ ] Unit tests
- [ ] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

File: shared/js/background/message-handlers.js [hunk 1]
```diff
@@ -33,11 +33,8 @@ export function getBrowser () {
     return browserName
 }
 
-export function firePixel (fireArgs) {
-    if (fireArgs.constructor !== Array) {
-        fireArgs = [fireArgs]
-    }
-    return pixel.fire.apply(null, fireArgs)
+export function submitBrokenSiteReport (brokenSiteArgs) {
+    return brokenSiteReport.fire.apply(null, brokenSiteArgs)
```
Comment (jonathanKingston) [inline shared/js/background/message-handlers.js:37]: This is needed to proxy from the UI as a message handler into the broken site report class.

Comment (jonathanKingston) [reply shared/js/background/message-handlers.js:37]: There's only one call site and it's always an array so there's no point supporting the other case now.

File: shared/js/background/message-handlers.js [hunk 2]
```diff
@@ -33,11 +33,8 @@ export function getBrowser () {
     return browserName
 }
 
-export function firePixel (fireArgs) {
-    if (fireArgs.constructor !== Array) {
```
Comment (kdzwinel) [inline shared/js/background/message-handlers.js:37]: any idea why we had this check before?

File: unit-test/background/pixel.es6.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
```
Comment (kdzwinel) [inline unit-test/background/pixel.es6.js:1]: hm, since we changed the original file being testsed here, I think we should also rename the test file

**Review by kdzwinel — COMMENTED** [review]
Thanks for doing this! I left two comments, but other than that it looks and works great.

File: unit-test/background/pixel.es6.js (hunk 3 above)
Comment (jonathanKingston) [reply unit-test/background/pixel.es6.js:1]: Yeah I was in two minds about this (as it's only testing pixels and not the whole thing), I think I agree :)

File: shared/js/background/message-handlers.js (hunk 2 above)
Comment (jonathanKingston) [reply shared/js/background/message-handlers.js:37]: `firePixel('eph')` was used in a few places for example and it also supported the `firePixel(['eph', ...])`

**Review by kdzwinel — APPROVED** [review]
Nicely done!

**Pull request merged**